### ISSUE TITLE
Missing lightlevel attribute at BOSCH Motion sensor

### ISSUE THREAD

**Issue by TribunMaximus** [BoschSmartHome/bosch-shc-api-docs#41]
Is it possible to include the lightlevel information into the api so that it can be changed without the need to use the BOSCH app?

**Comment by wsirhc** [comment]
Regarding [https://community.home-assistant.io/t/bosch-smart-home/115864/99?u=chris.w](HA-Forum), here are some curl calls which worked for me
**example multilevelswitchstate get value**
```
curl --location --request GET 'https://192.168.180.103:8444/smarthome/devices/hdm:ZigBee:0123456789abcdef/services/MultiLevelSwitch/state' \
--header 'Content-Type: application/json' \
--header 'api-version: 1.0'
```

**example multilevelswitchstate value change**
```
curl --location --request PUT 'https://192.168.0.10:8444/smarthome/devices/hdm:ZigBee:0123456789abcdef/services/MultiLevelSwitch/state' \
--header 'Content-Type: application/json' \
--header 'api-version: 1.0' \
--data-raw '{
    "@type": "multiLevelSwitchState",
    "level": "10"
}'

```

**Comment by tschamm** [comment]
I think things were mixed up a bit in this issue.
The topic of this issue is on the Motion Sensor, and the current API does not expose an option to control the lightlevel (sensitivity) of the motion sensor.
@wsirhc The MultiLevelSwitch defines the "brightness level" of a light.

**Comment by sebhard** [comment]
@tschamm, do you mean the brightness sensitivity of the motion light service of a motion detector?

**Comment by tschamm** [comment]
@sebhard correct, in the issue we asked for an option to control the motion light service, if this is possible to set via API.

**Comment by sebastianharder** [comment]
@tschamm there is an API for that but I have to clarify if we want to make it public.

**Comment by sebastianharder** [comment]
@tschamm unfortunately, restricted clients are not allowed to make configurations like that and we don't plan to change this any time soon. Sorry!

**Comment by tschamm** [comment]
Thanks for the clarification!

**Comment by Purii** [comment]
Is there a possibility to just read to current `lightlevel` (lx)?
